game-of-life/README.md

```diff
@@ -1,17 +1,31 @@
-Write a cellular automata that models the [Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life)
+# Game of Life
 
-Requirement Cards:
+## Background 
 
-1. A cell can be made "alive"
+The Game of Life is a cellular automaton. 
```

Comment:
I think we could drop this line and just have 'the game of life is discussed in detail...